android/src/main/java/com/facebook/reactnative/androidsdk/Utility.java

```diff
@@ -168,10 +168,6 @@ public static GameRequestContent buildGameRequestContent(ReadableMap gameRequest
     public static ShareLinkContent buildShareLinkContent(ReadableMap shareLinkContent) {
         ShareLinkContent.Builder contentBuilder = new ShareLinkContent.Builder();
         contentBuilder.setContentUrl(Uri.parse(shareLinkContent.getString("contentUrl")));
-        String url = getValueOrNull(shareLinkContent, "imageUrl");
-        contentBuilder.setImageUrl(url != null ? Uri.parse(url) : null);
-        contentBuilder.setContentDescription(getValueOrNull(shareLinkContent, "contentDescription"));
-        contentBuilder.setContentTitle(getValueOrNull(shareLinkContent, "contentTitle"));
```

Comment:
These are still referenced in our docs though https://github.com/thebergamo/react-native-fbsdk-next#share-dialogs - how should people adapt?

Reply:
Looks like those 3 parameters that were already marked as deprecated have now been deleted, not much people can do beside stop using them right?

Reply:
Okay - can you add a commit with changes to this page https://github.com/thebergamo/react-native-fbsdk-next#share-dialogs to reflect what is possible now?

Other than that this is probably good to go then, I only use login myself so I'm not a great test, although the example app does at least attempt a share. If there are problems others can propose fix ups I think, it's open source...

Reply:
Awesome, just updated it. Thank you so much for your help Mike